opendcs-rest-api/src/main/java/org/opendcs/odcsapi/sec/basicauth/TokenAuthenticatorFilter.java

```diff
@@ -0,0 +1,83 @@
+/*
+ *  Copyright 2024 OpenDCS Consortium
```

Comment:
FYI, this should actually say `and it's Contributors` now. Don't really need to worry about it right no but keep it in mind.